src/protocol/tx-validity.md

```diff
@@ -83,6 +83,13 @@ If this check passes, the UTXO ID `(txID, outputIndex)` fields of each contract
 For each asset ID `asset_id` in the input and output set:
 
 ```py
+def gas_to_fee(gas, gas_price) -> int:
```

Comment:
Is Python just used as the "pseudo-code" for defining behavior?

Reply:
That is essentially my understanding, yes!